activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeBPMNSignalReceivedIT.java

```diff
@@ -189,4 +190,28 @@ public void shouldGetSignalReceivedEventWithVariables() {
         assertThat(event.getEntity().getSignalPayload().getVariables().size()).isEqualTo(1);
         assertThat(event.getEntity().getSignalPayload().getVariables().get("signal-variable")).isEqualTo("test");
     }
+    
+    
+    @Test
+    public void shouldThrowSignalAndCompleteProcess() {
+    	
+        //given
+        securityUtil.logInAs("salaboy");
+
+        //when
+        ProcessInstance process = processRuntime.start(ProcessPayloadBuilder.start()
+                                                               .withProcessDefinitionKey("signalThrowEventProcess")
+                                                               .withVariable("name",
+                                                                             "peter")
+                                                               .build());
+        
+        //then
+        assertThat(process.getStatus()).isEqualTo(ProcessInstanceStatus.COMPLETED);
```

Comment:
@CTI777 @erdemedeiros what is this test testing?
We should at least check that the BPMN elements (events) for the throw signal is emitted right?